portal/src/locale-data/en.json

```diff
@@ -395,6 +395,8 @@
   "UserDetails.account-security.primary": "Primary Authenticators",
   "UserDetails.account-security.secondary": "2-Factor Authenticators",
   "UserDetails.account-security.secondary.add": "Add 2FA",
+  "UserDetails.account-security.primary.password.empty": "No Primary Authenticator yet.",
```

Comment:
Shouldn't this message be `No Primary Password yet.`?

Reply:
It looks like this so I think it should be primary authenticator. Let me update the message id.

<img width="683" alt="Screenshot 2025-07-03 at 1 05 00 PM" src="https://github.com/user-attachments/assets/d460e885-5cd3-48e6-b3ab-d6b940432ae5" />